interfaces/cython/cantera/composite.py

```diff
@@ -63,7 +63,7 @@ class Solution(ThermoPhase, Kinetics, Transport):
     directly in Python::
 
         spec = ct.Species.listFromFile('gri30.yaml')
-        rxns = ct.Reaction.listFromFile('gri30.yaml')
+        rxns = ct.Reaction.listFromFile('gri30.yaml', gas)
```

Comment:
Can you update this to give a complete example, where you first initialize a gas with the species but no reactions, then create the list of reactions? Right now, `gas` is used before it is defined.

Reply:
I pushed an update defining `spec_gas` explicitly from the species (plus a sentence stating the requirement). It looks a bit contrived, but the example runs without error. What do you think?